src/common.ts

```diff
@@ -521,15 +527,19 @@ export function toggleSection(sectionName: string = '', button: HTMLButtonElemen
 export function researchTeamDropdown(inputElement: HTMLSelectElement = globalElements.input.researchTeam as HTMLSelectElement, civ = pageData.civShort) {
 	if (!inputElement) return;
 	const prevSelect = inputElement.value;
-	const teams = ['', 'GHGS', 'GHEC', 'GHSH', 'GHDF', 'GHBG', 'GHSL', 'GHTD', 'HBS'];
+	const teams = ['none'];
```

Comment:
You could just remove the whole researchteam input field from the HTML, it should then default to the civ on its own. Or even better, you just hardcode the civ into the researchteam infobox parm.